modules/coursier/shared/src/main/scala/coursier/parse/ReconciliationParser.scala

```diff
@@ -0,0 +1,34 @@
+package coursier.parse
+
+import coursier.core.{Module, ModuleName, Organization, Reconciliation}
+import coursier.util.{ModuleMatcher, ModuleMatchers, ValidationNel}
+import coursier.util.Traverse._
+
+object ReconciliationParser {
+  def reconciliation(input: Seq[String], scalaVersionOrDefault: String): Either[List[String], Seq[(ModuleMatchers, Reconciliation)]] = {
```

Comment:
Return a `ValidationNel` here rather than an `Either`?

Reply:
Addressed in https://github.com/coursier/coursier/pull/1293/commits/7d8966f262200bff55a5bef70b19af11af0f7931